modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -1,5 +1,11 @@
 definition {
 
+	macro click {
+		Click.mouseDownMouseUp(
+			key_fieldText = "${button}",
+			locator1 = "PRMMDFRequest#TEXT_BUTTON");
+	}
+
```

Comment:
Hi @juliosilveira77, could you please rename this macro to: `activityCollapseButton`?